js/interface/map-panel.js

```diff
@@ -56,7 +131,29 @@ Fliplet.InteractiveMap.component('map-panel', {
         }
       })
 
-      window.filePickerProvider.then((result) => {
+      window.filePickerProvider.then(result => {
+        if (this.shouldKeepMarkers) {
+          let newImageWidth = result.data[0].size[0]
```

Comment:
Do we need to be concerned with what if `data` is empty, i.e. user has not selected any files?

Reply:
If the user did not select any files, then no modal window will appear, therefore this code would not run.